src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigatorContext.java

```diff
@@ -63,4 +68,18 @@ void setTeamSlug(String teamSlug) {
     public String getTeamSlug() {
         return teamSlug;
     }
+
+    /**
+     * @return True if archived repositories should be ignored, false if they should be included.
+     */
+    public boolean isExcludeArchived() {
+        return excludeArchived;
+    }
+
+    /**
+     * @param excludeArchived Set true to exclude archived repositories
+     */
+    public void setExcludeArchived(boolean excludeArchived) {
+        this.excludeArchived = excludeArchived;
+    }
```

Comment:
I think we do this differently now that we're using Traits. I have to dig a bit.

Reply:
Nope, my bad, this is fine.